MapboxNavigation/NavigationMapView.swift

```diff
@@ -693,12 +684,10 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
             } else {
                 
                 arrowStroke.minimumZoomLevel = minimumZoomLevel
-                arrowStroke.lineCap = MGLStyleValue(rawValue: cap)
-                arrowStroke.lineJoin = MGLStyleValue(rawValue: join)
-                arrowStroke.lineWidth = MGLStyleValue(interpolationMode: .exponential,
-                                                      cameraStops: routeLineWidthAtZoomLevels.multiplied(by: 0.80),
-                                                      options: [.defaultValue : MGLConstantStyleValue<NSNumber>(rawValue: 1.5)])
-                arrowStroke.lineColor = MGLStyleValue(rawValue: .defaultArrowStroke)
+                arrowStroke.lineCap = NSExpression(forConstantValue: cap)
```

Comment:
`NSExpression(forConstantValue: "cap")` or `NSExpression(format: "'cap'")` is also allowed now, although what you’ve got is also perfectly fine if you want compile-time validation.